modules/storage-persistent-storage-nfs-provisioning.adoc

```diff
@@ -84,5 +86,5 @@ rather act as labels to match a PV to a PVC.
 ----
 $ oc get pvc
 NAME         STATUS   VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
-nfs-claim1   Bound    pv0001   5Gi        RWO            gp2            2m
+nfs-claim1   Bound    pv0001   5Gi        RWO                           2m
```

Comment:
Not necessarily in scope for this PR, but a couple of things that should be addressed either along with this PR or as a separate PR/BZ:
- The [OpenShift Docs guidelines](url) require that we split commands and output into separate code blocks.
- In the first callout note, the first clause could be removed: "As mentioned above for PVs," both because it isn't important and because ISG advises against using location words such as "above" and "below" and replace with "previous" or "following", or omit altogether, which in this case seems best.

Let me know if you'd like me to open a separate BZ for these 2 items or if you'd rather apply the changes in this PR. Thanks!

Reply:
@bobfuru not required as these changes are not technical.